api/MutationObserverInit.json

```diff
@@ -113,7 +113,8 @@
               "version_added": "14"
             },
             "ie": {
-              "version_added": "11"
+              "version_added": "11",
+              "notes": "On IE11 using <code>attributeFilter</code> without setting <code>attributes: true</code> will throw a syntax error."
```

Comment:
```suggestion
              "notes": "Internet Explorer requires <code>attributes: true</code> when using <code>attributeFilter</code>.  If <code>attributes: true</code> is not present, Internet Explorer will throw a syntax error."
```

Reply:
I updated this with my latest commit. Resolving